pkg/resources/statefulsets/minio-statefulset.go

```diff
@@ -888,14 +888,15 @@ func getInitContainer(t *miniov2.Tenant, operatorImage string) v1.Container {
 		VolumeMounts: []corev1.VolumeMount{
 			CfgVolumeMount,
 		},
+		SecurityContext: poolContainerSecurityContext(pool),
```

Comment:
At the code level, this is really fine, but there is a potential problem with upgrades. If the previous tenant did not have this field, and now because you did not make a judgment, the default added, there will be operator error situation, have you tested it?

Reply:
@jiuker I havent tested the upgrade scenario. Let me do that as update.